### PR TITLE
update supported version for j9 15+

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by sxa — CHANGES_REQUESTED** [review]
While this appears to work, I think this needs some documentation/comments added to make it clear why `11` isn't included in the list along with `8` and `15` otherwise it's rather non-obvious to someone looking at this why and the reason is somewhat hidden in the code in `getSupportedVersion` so _at the very least_ it needs a sentence above the function indicating how the string is selected.

File: src/js/common.js [hunk 1]
```diff
@@ -63,7 +63,23 @@ module.exports.getBinaryExt = (searchableName) => lookup[searchableName].binaryE
 module.exports.getInstallerExt = (searchableName) => lookup[searchableName].installerExtension;
 
 // gets the Supported Version WITH PATH when you pass in 'searchableName'
-module.exports.getSupportedVersion = (searchableName) => lookup[searchableName].supported_version;
+module.exports.getSupportedVersion = (searchableName) => {
+  let supported_version = lookup[searchableName].supported_version;
+  if (typeof supported_version === 'object') {
```
Comment (karianna) [inline src/js/common.js:68]: `supported_version === 'object'` - This is why I weep at dynamic typing ;-).  V2 of the website - I vote we go Typescript

Comment (gdams) [reply src/js/common.js:68]: agreed :D

**Comment by gdams** [comment]
> While this appears to work, I think this needs some documentation/comments added to make it clear why 11 isn't included in the list along with 8 and 15 otherwise it's rather non-obvious to someone looking at this why and the reason is somewhat hidden in the code in getSupportedVersion so at the very least it needs a sentence above the function indicating how the string is selected.

@sxa updated, comments in JSON are always horrible but it's the best that I can do

**Comment by sxa** [comment]
> @sxa updated, comments in JSON are always horrible but it's the best that I can do

I was more thinking about commenting the js function to be honest, although certainly a comment in there is welcome (even though saying it's >= isn't entirely accurate since 16>=8, but any clearer wording would be longer unless we could link into the parsing function)

**Comment by sxa** [comment]
Also (I don't know why this randomly came into my head at 5AM) is there a "booboy-trap" here for anyone updating or trying to use this new logic whereby if the JVM version numbers are not in ascending order it won't work?

**Review by sxa — APPROVED** [review]
Thanks - we should consider if we can make the logic a bit more maintainable/reliable to avoid people accidentially introducing errors in the JSON in the future though

**Pull request merged**